pallets/mips/src/lib.rs

```diff
@@ -544,6 +575,11 @@ impl<T: Trait> Module<T> {
             priority.clone(),
             proposal_hash.clone(),
         ));
+
+        // If committee size is too small, enact it.
+        if T::GovernanceCommittee::member_count() < 2 {
+            Self::prepare_to_dispatch(proposal_hash.clone());
```

Comment:
I think this logic needs to be in committee.rs to correctly evalulate the GC quorum when there is only a single GC member. As written here, the GC member won't need to vote at all, and every referenda is automatically passed?

Reply:
This is for a special case. Since GC members can create referendums directly, if the committee is composed of a single member, it's enacted immediately. This way single GC member need not create a MIP to enact the referendum they just created.

Without this if block, the process gets a bit tedious for the single member for the same outcome:
1. Member creates a referendum
2. Member creates a MIP proposal to `enact_referendum` with hash from step 1
3. Committee executes `enact_referendum` it because member count is < 2

Reply:
Ah OK - that makes sense. Do we not need some logic in committee anyway so that in the case of a normal GC vote a single member can still function? i.e. checking whether the threshold is passed during the proposal function as well in the vote function.

Reply:
The original verison had the logic but went missing. I've added the checks in `committee.rs` to execute a proposal when member count < 2. Also added a test case in `committee_test.rs` to ensure proposal is executed when a single member committee makes a proposal.

Reply:
@adamdossa there's an edge case to consider: 
1. Say Alice, Bob, Charlie are committee members. 
2. Alice makes a proposal. 
3. Bob and Charlie quit.
4. Alice can't quit since she's the last remaining member. 
The proposal Alice made remains in queue.

Reply:
OK. I guess in this case Alice can just make a new proposal, which would now be automatically approved as she is now the only remaining member. If this is the case then I think we can live with this.

The alternative would be to iterate over all proposals every time the GC membership changes and approve any which now meet the new implied threshold - I think this overkill at the moment, provided the first statement above holds.